aten/src/ATen/native/cuda/jit_utils.cpp

```diff
@@ -893,6 +893,8 @@ void codegenOutputQuery(
     max_dev_version = CUDAVersion(7, 5);
   } else if (nvrtc_version == CUDAVersion(11, 0)) { // 11.0 supports 3-8.0
     max_dev_version = CUDAVersion(8, 0);
+  } else if (nvrtc_major == 11 && nvrtc_minor < 8) {
```

Comment:
```suggestion
  } else if (nvrtc_major == 11 && nvrtc_minor =< 8) {
```

Reply:
BUt versions less than 11.8 support only up to sm_86, so it's like this on purpose, or is my reasoning wrong?

Reply:
I thought nvrtc from 11.8 does not support sm_89, as well, does it?

Reply:
It should, according to #87436 and the docs.